### PR TITLE
Processing example

### PR DESCRIPTION
**Pull request by Siegrift**
https://api3dao.atlassian.net/browse/AN-731

I've wanted to make just a single example, but none of the CoinGecko endpoints I looked didn't offer a suitable use case for both. So I've created:
- pre-processing example - expects an UNIX timestamp and uses that to convert it to the timestamp expected by CoinGecko
- post-processing example - calls endpoint of multiple coins and computes the average of their prices and percentage change.

### CODE REVIEW

File: .changeset/eighty-buttons-kick.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-examples': minor
```
Comment (Siegrift) [inline .changeset/eighty-buttons-kick.md:2]: I have modified spelling in other examples, but that's not worth mentioning in changelog.

**Review by amarthadan — APPROVED** [review]
Haven't run these, I'm leaving this for @kolenic-martin but the code looks good to me :+1:

**Comment by kolenic-martin** [comment]
Hey, @Siegrift, I've run into an issue when attempting to run the new example in the end:

```
(node:33) UnhandledPromiseRejectionWarning: Error: Failed to parse config file
    at readConfig (/app/src/config/index.js:55:15)
    at loadTrustedConfig (/app/src/config/index.js:69:21)
    at loadConfig (/app/src/workers/local-handlers.js:85:43)
    at Object.<anonymous> (/app/src/workers/local-handlers.js:93:30)
    at step (/app/src/workers/local-handlers.js:56:23)
    at Object.next (/app/src/workers/local-handlers.js:37:53)
    at /app/src/workers/local-handlers.js:31:71
    at new Promise (<anonymous>)
    at __awaiter (/app/src/workers/local-handlers.js:27:12)
    at Object.startCoordinator (/app/src/workers/local-handlers.js:88:12)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:33) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:33) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

The error has been thrown three times as I attempted to ignore it and hope for the best. In the end, the request timed out. I only ran Airnode locally but used Rinkeby as the testing blockchain.

**Comment by aquarat** [comment]
This is great @Siegrift

**Pull request merged**